packages/client/src/Views/ChannelTabsContainer/helpers/tabRender.tsx

```diff
@@ -20,6 +36,10 @@ const tabRender = (state: PlayerUi): (node: TabNode) => void  => {
 
       channel = matchedChannel && matchedChannel.length > 1 ? matchedChannel[1] : undefined
 
+      setTimeout(() => {
+        addMenuItemMsgCount(`unread-${channel.unreadMessageCount}`)
+      })
```

Comment:
does the above line need to be modified to return `9plus` if there are 9 more unread messages?

Reply:
ah not yet Ian, do you want to show 9+ or the real number unread message count here? e.g: (12) or (23)?

Reply:
9+ is fine, thanks.